test/integration/disruption/disruption_test.go

```diff
@@ -200,23 +199,6 @@ func TestEmptySelector(t *testing.T) {
 		createPDBFunc          func(ctx context.Context, clientSet clientset.Interface, name, nsName string, minAvailable intstr.IntOrString) error
 		expectedCurrentHealthy int32
 	}{
-		{
-			name: "v1beta1 should not target any pods",
-			createPDBFunc: func(ctx context.Context, clientSet clientset.Interface, name, nsName string, minAvailable intstr.IntOrString) error {
-				pdb := &v1beta1.PodDisruptionBudget{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: name,
-					},
-					Spec: v1beta1.PodDisruptionBudgetSpec{
-						MinAvailable: &minAvailable,
-						Selector:     &metav1.LabelSelector{},
-					},
-				}
-				_, err := clientSet.PolicyV1beta1().PodDisruptionBudgets(nsName).Create(ctx, pdb, metav1.CreateOptions{})
```

Comment:
we still need to test compatibility with v1beta1 PDB objects persisted in etcd... consider alternative ways to persist these directly into etcd if we're no longer serving the v1beta1 endpoints (see use of the GetEtcdClients() helper in other integration tests used to read/write directly to etcd)